src/panels/ClusterPropertiesPanel.ts

```diff
@@ -64,6 +65,29 @@ export class ClusterPropertiesDataProvider implements PanelDataProvider<"cluster
         }
     }
 
+    private async _handleAbortClusterOperation(webview: MessageSink<ToWebViewMsgDef>) {
+        try {
+            await this._readAndPostClusterProperties(webview);
```

Comment:
I see this line is in the agent pool abort method, too, but I can't remember why we added it, and can't see a reason for having it. Do you remember?

Reply:
Yeah, I think it was their for status update, but let me see removing will fix the enable/disable button thing. 🤷🏽‍♂️☕️☕️